src/index.js

```diff
@@ -1,37 +1,117 @@
 import './style.css';
+import {
+  renderToDo, edit,
+} from './modules/functionality.js';
 
-const activities = [
-  {
-    description: 'Firts',
-    completed: false,
-    id: 1,
-  },
-  {
-    description: 'Second',
-    completed: false,
-    id: 2,
-  },
-  {
-    description: 'Thrid',
+const form = document.getElementById('list-form');
+const containerTask = document.getElementById('toDo-list');
+const newToDo = document.getElementById('imput-task');
+let toDos = [];
+
+// function add taks
+const addTask = () => {
+  const toDoValue = newToDo.value;
+  const emptyToDo = toDoValue === '';
+
+  if (emptyToDo) return;
+
+  const task = {
+    value: toDoValue,
     completed: false,
-    id: 3,
-  },
-];
-
-const containerTask = document.getElementById('task-list');
-
-const showTaskList = () => {
-  activities.forEach((task) => {
-    const card = document.createElement('div');
-    card.classList = 'task-description';
-    card.innerHTML = `
-    <div class="task-text">
-    <input type="checkbox">
-    <p class="task-text">${task.description}</p>
-    </div>
-    <i class="fa-solid fa-ellipsis-vertical"></i>`;
-    containerTask.appendChild(card);
-  });
+    id: toDos.length + 1,
+  };
+
+  newToDo.value = '';
+  toDos.push(task);
+  localStorage.setItem('todos', JSON.stringify(toDos));
+
+  renderToDo(); // llamamos a la función renderToDo para mostrar el nuevo elemento en la pantalla
 };
 
-showTaskList();
\ No newline at end of file
+// const addTask = () => {
+//   const toDoValue = newToDo.value;
+//   const emptyToDo = toDoValue === '';
+
+//   if (emptyToDo) return;
+//   const task = {
+//     value: toDoValue,
+//     completed: false,
+//     id: toDos.length + 1,
+//   };
+//   newToDo.value = '';
+//   toDos.push(task);
+//   localStorage.setItem('todos', JSON.stringify(toDos));
+// };
```

Comment:
- [ ] Please make sure you remove commented code from your project to make your code clean. :pray: You should not push commented code to your repository. You can read more about it [here](https://www.freecodecamp.org/news/code-comments-the-good-the-bad-and-the-ugly-be9cc65fbf83/)

Reply:
thanks!!! is it on the add and remove branch? cause I cant find it!